src/main/resources/application.properties

```diff
@@ -54,8 +54,10 @@ notification.url=${BASE_URL}
 cors.allowedOrigins=${CORS_ALLOWED_ORIGINS:}
 cors.allowedMethods=${CORS_ALLOWED_METHODS:}
 
-email.urlToViewBinCard=${BASE_URL}/#!/stockmanagement/stockCardSummaries/{0}
-email.urlToInitiateRequisition=${BASE_URL}/#!/requisitions/initiate?facility={0}&program={1}&emergency={2}&supervised={3}
+publicUrl=${PUBLIC_URL:-${BASE_URL}}
+
+email.urlToViewBinCard=${PUBLIC_URL:-${BASE_URL}}/#!/stockmanagement/stockCardSummaries/{0}
```

Comment:
Can't we just use publicUrl property value here?

Reply:
I am not sure if this is possible but of course, we should check the possibility

Reply:
Actually, it is: https://stackoverflow.com/questions/36136874/spring-properties-that-depend-on-other-properties

I have changed this to use the publicUrl property.